collectors/hra/parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .record import Record
+from datetime import date
+import logging
+import six
+logger = logging.getLogger(__name__)
+
+
+def parse_response(response, endpoint, from_date, to_date, errors, success, conn):
```

Comment:
I suppose you mean I should keep `parser.py` only for parsing functionality, which seems right... It was written like this because I just copied my parsing function that I had written when everything was in `collecot.py`. I will move logging back to `collector.py` and also no need to pass the `conn`.